src/blueapi_config.yaml

```diff
@@ -0,0 +1,7 @@
+env:
```

Comment:
Rather than `src`, should this go in `config`?

Reply:
Initially I would have preferred it to go in the top level directory, i.e. even before src/ ... But perhaps for pypi purposes that isn't a good idea? I'm not really sure.

I don't know how you're expecting blueapi to be used - i.e. thought it would just be in a container.